config/helm/appmesh-controller/README.md

```diff
@@ -379,14 +379,16 @@ Parameter | Description | Default
 `init.image.tag` | Route manager image tag | `<VERSION>`
 `stats.tagsEnabled` |  If `true`, Envoy should include app-mesh tags | `false`
 `stats.statsdEnabled` |  If `true`, Envoy should publish stats to statsd endpoint @ 127.0.0.1:8125 | `false`
-`stats.statsdAddress` |  DogStatsD daemon IP address | `127.0.0.1`
-`stats.statsdPort` |  DogStatsD daemon port | `8125`
+`stats.statsdAddress` |  DogStatsD daemon IP address. This will be overridden if `stats.statsdSocketPath` is specified | `127.0.0.1`
+`stats.statsdPort` |  DogStatsD daemon port. This will be overridden if `stats.statsdSocketPath` is specified | `8125`
+`stats.statsdSocketPath` | DogStatsD Unix domain socket path. If not specified and if `stats.statsdEnabled` is set to `true` then it will default to DogStatsD daemon IP address port [default: `127.0.0.1:8125`] | None
 `cloudMapCustomHealthCheck.enabled` |  If `true`, CustomHealthCheck will be enabled for CloudMap Services | `false`
 `cloudMapDNS.ttl` |  Sets CloudMap DNS TTL | `300`
 `tracing.enabled` |  If `true`, Envoy will be configured with tracing | `false`
 `tracing.provider` |  The tracing provider can be x-ray, jaeger or datadog | `x-ray`
 `tracing.address` |  Jaeger or Datadog agent server address (ignored for X-Ray) | `appmesh-jaeger.appmesh-system`
-`tracing.port` |  Jaeger or Datadog agent port (ignored for X-Ray) | `9411`
+`tracing.port` |  X-Ray, Jaeger or Datadog agent port | `9411`
+`tracing.samplingRate` | X-Ray tracer sampling rate. Value can be a decimal number between 0 and 1.00 (100%)  | `0.05`
```

Comment:
The precision is strictly 2 decimal point. which will translate to percentage in number.
If higher precision is set, it will be rounded to nearest 2 decimal point float. Ex: `0.049 ~> 0.05`

Reply:
Saying that we don't stop user from setting a float value with higher precision. It will be handle by math in pkg/inject/sidecar_builder.go L103 and converted to float with just 2 decimal precision after .